README.md

```diff
@@ -107,7 +107,7 @@ workdir: ~/torch_examples
 # Commands to be run before executing the job.
 # Typical use: pip install -r requirements.txt, git clone, etc.
 setup: |
-  pip install torch torchvision
```

Comment:
There's a bunch of [other occurrences](https://github.com/search?q=repo%3Askypilot-org%2Fskypilot%20pip%20install%20torch%20torchvision&type=code) of `pip install torch torchvision`, would we need to update those too? Or perhaps we can find a more robust fix (e.g., updating cloud image)

Reply:
Yea, we need to fix those too. This PR tends to be a temporary fix for the problem in case someone else is trying yaml in readme. Maybe we can have a more robust fix, like updating cloud image in a later PR. : )